public/index.html

```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <script src="%PUBLIC_URL%/env.js?version=123"></script>
```

Comment:
I'm not sure I like this solution. Most of the users (at least at the moment), aren't using docker so they won't need that script.

Reply:
did this to turn on & off the config loading from external storage like (s3). Please suggest any alternative if this is not acceptable.

Reply:
What about having another `index.html` file for docker?